src/schema/v2/me/index.ts

```diff
@@ -630,7 +630,8 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
       },
     },
     submissionsConnection: submissionsConnection,
-    // genomic recommendation
+    // DEPRECATED: This genomic recs loader is no longer in use.
+    // Use the artworkRecommendations connection to Vortex instead.
     recommendedArtworks: {
       type: artworkConnection.connectionType,
       args: pageable({
```

Comment:
```suggestion
    recommendedArtworks: {
       deprecationReason: "This genomic recs loader is no longer in use. Use the artworkRecommendations instead.",
      type: artworkConnection.connectionType,
      args: pageable({
```

Reply:
makes sense... thanks for the suggestion!